persistence/src/main/scala/hmda/persistence/institutions/SubmissionPersistence.scala

```diff
@@ -29,16 +29,23 @@ object SubmissionPersistence {
       event match {
         case SubmissionCreated(s) =>
           SubmissionState(s +: submissions)
+
         case SubmissionStatusUpdated(id, status) =>
-          val x = submissions.find(x => x.id == id).getOrElse(Submission())
+          val x: Submission = submissions.find(x => x.id == id).getOrElse(Submission())
```

Comment:
Is this type needed here? I think type inference takes care of figuring out this type, and I don't find this code confusing without the declaration, it's quite obvious that `x` is going to be of type `Submission`